_events/2022/2022-08-dei-speaker-series.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Diversity in Open-source Software
+subtitle: US-RSE DEI-WG Speaker Series
+expires: 2022-08-31 17:00:00 ET
+event_date: "August 31, 2022"
+layout: event
+duration: 60
+repeated: false
+category: dei
+time:
+    - - start: 2022-08-31T20:00:00Z
+        end: 2022-08-31T21:00:00Z
+---
+
+
+
+## US-RSE DEI-WG Speaker Series Info
+
+The US-RSE DEI-WG is working to bring to the US-RSE community quarterly visits
+from RSE professionals with vested interests or research into DEI in the RSE
+community at large. Our community pledges to continually educate ourselves
+through research on best practices focused on diversity, equity, and inclusion.
+US-RSE is committed to providing an inclusive environment with equitable
+treatment for all and to promoting and encouraging diversity throughout the RSE
+community. We encourage everyone to suggest speakers and topics and are open to
+different formats and timelines of the events. Feedback is much appreciated.
+
+
+## Event
+
+The US-RSE Diversity, Equity, and Inclusion Working Group (DEI-WG) is proud to
+welcome Vandana Singh to present his talk "Diversity in Open-source Software."
```

Comment:
Further down they become a "her", unless this is some diversity thing?

Reply:
Again thank you for catching this!